<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>11</y>
     <width>651</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="orderTable">
      <property name="dragDropOverwriteMode">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>상품명</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>수량</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>금액</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="sellList">
      <item>
       <property name="text">
        <string>맹독 젤리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>기술의 역사</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>320</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>총결제금액</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="totalPrice">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>받은 금액</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="getMoney">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>60000</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>거스름돈</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="changeMoney">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>427</y>
     <width>335</width>
     <height>170</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="card_btn">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>81</height>
       </size>
      </property>
      <property name="text">
       <string>카드 결제</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="cash_btn">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>81</height>
       </size>
      </property>
      <property name="text">
       <string>현금 결제</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="reset">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>81</height>
       </size>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="soldBtn">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>81</height>
       </size>
      </property>
      <property name="text">
       <string>결제</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Atype">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>81</height>
       </size>
      </property>
      <property name="text">
       <string>A타입 영수증</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Btype">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>81</height>
       </size>
      </property>
      <property name="text">
       <string>B타입 영수증</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
